src/server/common/model/answer/confirmation/confirmation.test.js

```diff
@@ -52,70 +52,61 @@ describe('#ConfirmationModel', () => {
   describe('toState', () => {
     it('should return an object with the correct data (confirm only)', () => {
       const confirmation = new ConfirmationAnswer({ confirmation: ['confirm'] })
-      expect(confirmation.toState()).toEqual({ confirmation: ['confirm'] })
+      expect(confirmation.toState()).toEqual(['confirm'])
```

Comment:
These tests need cutting way back, since they replicate the tests from the base class they inherit from